src/Symfony/Component/HttpFoundation/HeaderUtils.php

```diff
@@ -289,6 +289,6 @@ private static function groupParts(array $matches, string $separators, bool $fir
             $parts[] = '' === $separators ? self::unquote($matches[0][0]) : self::groupParts($matches, $separators, false);
         }
 
-        return $parts;
+        return array_filter($parts);
```

Comment:
well, shouldn't we do a strict comparision, so that we don't filter out he `"0"` string? is that a possible situation?

Reply:
makes sense, I have rewritten the patch a bit